vite.config.ts

```diff
@@ -13,7 +13,7 @@ export default defineConfig(() => ({
       formats: ["es", "cjs"] as LibraryFormats[], // typescript is unhappy without this forced type definition
     },
     rollupOptions: {
-      external: ["react", "react-dom", "@measured/puck"],
+      external: ["react", "react-dom", "@measured/puck", "uuid"],
```

Comment:
We needed to externalize uuid? If we do that I don't think it'd work in a starter without the starter having uuid as a dep, though I could be wrong.

Reply:
Oh hmmm I did that b/c

[vite]: Rollup failed to resolve import "uuid" from "/Users/asanehisa/visual-editor/src/internal/utils/generateId.ts".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`

Reply:
yay tested + works fine in the pages visual editor starter (even without adding uuid as dep to that starter)